engine/src/main/java/org/destinationsol/game/console/ShellInputHandler.java

```diff
@@ -40,7 +43,7 @@ public ShellInputHandler() {
     }
 
     /**
-     * Registers a command with this input handler.
+     * Registers a command with this input handler. If the command is already registered throws a {@link SolException}
```

Comment:
What about adding a `@throws` javadoc magic word? Not sure how that is called, it's late for me to think any much more now

Reply:
Thank you, I changed it in the next commit like you asked.